tests/test_pipelines_automatic_speech_recognition.py

```diff
@@ -68,14 +68,14 @@ def test_torch_large(self):
             tokenizer="facebook/wav2vec2-base-960h",
             framework="pt",
         )
-        waveform = np.zeros((34000,))
+        waveform = np.linspace(0, 1, 34000, dtype=np.float32)
         output = speech_recognizer(waveform)
         self.assertEqual(output, {"text": ""})
 
         from datasets import load_dataset
 
-        ds = load_dataset("patrickvonplaten/librispeech_asr_dummy", "clean", split="validation")
-        filename = ds[0]["file"]
+        ds = load_dataset("patrickvonplaten/librispeech_asr_dummy", "clean", split="validation").sort("id")
```

Comment:
_Unrelated to this PR_: the new`datasets` version re-shuffled this dataset, so sorting is needed for reproducibility.